### ISSUE TITLE
Magic Workbench recipes don't use datapacks, so they can't be changed by modpack developers

### ISSUE THREAD

**Issue by aaronhowser1** [elucent/eidolon#125]
I unified all metals to use Thermal Foundation's, including Lead. That means I can't get Eidolon's Lead, and I can't change the Mind-Shielding Plate's recipe to use TF's Lead.

**Comment by Jerome226** [comment]
This is a know issue since at least 6 months.
Would suggest using https://www.curseforge.com/minecraft/mc-mods/eidolon-recipes if you want to have JSON support of Eidolon.

**Comment by Darkmega18** [comment]
it can't edit the existing ones. D:

**Comment by elucent** [comment]
Will be supported officially in 1.18.

**Comment by Darkmega18** [comment]
............................................
well that does me no good with balancing my 1.16.5 pack. but ok... :V

**Comment by elucent** [comment]
I don't really have the time to support both versions; as you can tell from the hiatus I barely have the time to work on Eidolon at all...so unfortunately a backport is unlikely.

**Comment by Darkmega18** [comment]
> I don't really have the time to support both versions; as you can tell from the hiatus I barely have the time to work on Eidolon at all...so unfortunately a backport is unlikely. :/

yeah, you work on mystical world etc with a bunch of other modders like noobanus etc don't you? Well... on the bright side the cauldron isn't exactly automatable with ease, atleast not obviously. so if people want to make high yield gunpowder or low cost gilded fruits to cheese out some EMC by hand, I guess they can. cause it takes a while to do, and you can stuff it up...

**Comment by Jerome226** [comment]
For people needing JSON Support in 1.16, there a unofficial mod that provides that:
https://www.curseforge.com/minecraft/mc-mods/eidolon-recipes
